qiita_pet/handlers/auth_handlers.py

```diff
@@ -165,11 +185,220 @@ def post(self):
 
     def set_current_user(self, user):
         if user:
-            self.set_secure_cookie("user", json_encode(user))
+            self.set_secure_cookie("user", json_encode(user), SameSite=None)
         else:
             self.clear_cookie("user")
 
 
+
+class KeycloakMixin(OAuth2Mixin):
+    _OIDC_CLIENT_ID = '%s' % qiita_config.oidc_clients['KEYCLOAK_1']['OIDC_CLIENT_ID']
```

Comment:
KEYCLOAK_1 is not necessarily in the config file. Kills my qiita at the moment

Reply:
Thank you for pointing that out. I forgot to add this change, but any placeholder value should work for starting the server.